### PR TITLE
Support conversion for glam 0.29

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

**Comment by pcwalton** [comment]
@dimforge Could we merge this soon? It's broken Bevy Rapier on the upcoming 0.15 release. Thank you :)

**Comment by waywardmonkeys** [comment]
@sebcrozet I've fixed the CI problem in #1454. If you merge that, I'll rebase this on top of it. It would be nice if there could be a 0.33.2 with this in it (and also clear out a couple of other PRs, like #1426, #1427, #1428).

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ documented here.
 
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## Unreleased
+
+### Added
+
+- Add the `glam-0.29` feature to enable conversion from/to types from `glam` v0.29.
```
Comment (sebcrozet) [inline CHANGELOG.md:12]:
```suggestion
- Add the `convert-glam029` feature to enable conversion from/to types from `glam` v0.29.
```

**Review by sebcrozet — APPROVED** [review]
Looking good, thanks!

**Pull request merged**